hypervisor/Makefile

```diff
@@ -42,7 +42,10 @@ CFLAGS += -mno-red-zone
 CFLAGS += -static -nostdinc -nostdlib -fno-common
```

Comment:
Shall we add '-static' only when CONFIG_RELOC is not defined, just like what you've done for the linking options?

Reply:
The GCC manual states that "-static":  On systems that support dynamic linking, this prevents linking with the shared libraries. On other systems, this option has no eﬀect.

So it seems to me that this doesn't matter to the hypervisor code relocation.

Reply:
Per LD manual, "-static" means exactly the same thing for the linker (i.e. "Do not link against shared libraries"), and the description of this option does not mention anything about relocation, either. Would you please describe the impact of "-static" on relocation a bit further so that everyone can understand why it is critical for the linker but not the compiler?

Reply:
Hi Junjie,

You are right. It's safer to wrap "CFLAGS += -static" with CONFIG_RELOC. Thanks for pointing it out.

Thanks,
Zide